src/FuncVersion.ts

```diff
@@ -18,14 +18,13 @@ export enum FuncVersion {
 export const latestGAVersion: FuncVersion = FuncVersion.v2;
 
 export async function promptForFuncVersion(message?: string): Promise<FuncVersion> {
-    const picks: IAzureQuickPickItem<FuncVersion | undefined>[] = [];
+    let picks: IAzureQuickPickItem<FuncVersion | undefined>[] = [
```

Comment:
I know why we list 2, 3, 1 but it still bothers me 🔢

Reply:
Open to 2, 1, 3 if you like that better